proto/ibc/core/channel/v1/upgrade.proto

```diff
@@ -6,6 +6,25 @@ option go_package = "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types";
 
 import "gogoproto/gogo.proto";
 import "ibc/core/client/v1/client.proto";
+import "ibc/core/channel/v1/channel.proto";
+
+// Upgrade is a verifiable type which contains the relevant information
+// for an attempted upgrade. It provides the proposed changes to the channel
+// end, the timeout for this upgrade attempt and the last packet sequence sent
+// to allow the counterparty to block sends after the upgrade has started.
+message Upgrade {
+  ModifiableUpgradeFields upgrade_fields   = 1 [(gogoproto.nullable) = false];
+  UpgradeTimeout          timeout          = 2 [(gogoproto.nullable) = false];
```

Comment:
wondering should we do something like:

```protobuf
UpgradeFields upgrade_fields            = 1 [(gogoproto.nullable) = false];
UpgradeTimeout upgrade_timeout          = 2 [(gogoproto.nullable) = false];
```

Reply:
Alternatively, to cut down a bit on the repetition of `upgrade`, it could be:

```
UpgradeFields  fields           = 1 [(gogoproto.nullable) = false];
UpgradeTimeout timeout          = 2 [(gogoproto.nullable) = false];
```

Reply:
definitely in favour of renaming these to be consistent.

I think I like `fields` and `timeout` since the type makes it very clear what it is already